configure.ac

```diff
@@ -2866,12 +2866,9 @@ if test "${with_mac}" != no; then
   test "${HAVE_MACGUI}" = yes && window_system=mac
 
   ## Specify the install directory
-  mac_appdir=`pwd`/mac
+  mac_appdir=`pwd`/carbon
```

Comment:
Given how confusing "Carbon" is, and that it's an API discontinued >10yrs ago, I think we may want to phase out that brand over time.  What's wrong with reusing `./mac/`?